ANTLR/src/Compiler/Lang.g4

```diff
@@ -19,16 +19,10 @@ stmt : expr SEMICOLON ;
 expr : val                                                                    #value
      | op=NEGATE val                                                          #uneryOp
      | left=expr op=POW right=expr                                            #binaryOp
-     | left=expr op=DIVIDE right=expr                                         #binaryOp
-     | left=expr op=MULTIPLY right=expr                                       #binaryOp
-     | left=expr op=PLUS right=expr                                           #binaryOp
-     | left=expr op=MINUS right=expr                                          #binaryOp
-     | left=expr op=LOGEQ right=expr                                          #binaryOp
-     | left=expr op=LOGNOTEQ right=expr                                       #binaryOp
-     | left=expr op=LOGLESS right=expr                                        #binaryOp
-     | left=expr op=LOGGREATER right=expr                                     #binaryOp
-     | left=expr op=LOGLESSOREQ right=expr                                    #binaryOp
-     | left=expr op=LOGGREATEROREQ right=expr                                 #binaryOp
+     | left=expr op=(DIVIDE|MULTIPLY) right=expr                              #binaryOp
+     | left=expr op=(PLUS|MINUS) right=expr                                   #binaryOp
+     | left=expr op=LOGRELATION right=expr                                    #binaryOp
```

Comment:
I would prefer being explicit in this case. `LOGRELATION` is very vague, especially as it precedes three other logical relation related productions.

Reply:
To elaborate, I would like to see the content of the "variable" directly here instead of having to look at line 63.
```suggestion
     | left=expr op=(LOGLESS | LOGGREATER | LOGLESSOREQ | LOGGREATEROREQ) right=expr                                    #binaryOp
```

Reply:
I don't think it's a problem that the line would get long in this case.